docs/features/gang-scheduling.md

```diff
@@ -0,0 +1,108 @@
+# Quickstart - Gang Scheduling
+
+## Introduction
+
+In this quickstart guide, we'll explore Gang Scheduling, a feature that ensures an "all or nothing" approach to scheduling pods.
+Gödel scheduler treats all pods under a job (PodGroup) as a unified entity during scheduling attempts.
+This approach eliminates scenarios where a job has "partially reserved resources", effectively mitigating resource deadlocks between multiple jobs and making it a valuable tool for managing complex scheduling scenarios in your cluster.
+This guide will walk you through setting up and using Gang Scheduling.
+
+## Local Cluster Bootstrap & Installation
+
+Before you begin with Gang Scheduling, ensure you have a local KIND cluster installed with Gödel. 
+For detailed instructions, refer to the [Cluster Setup Guide](kind-cluster-setup.md).
+
+## Using Gang Scheduling
+
+1. **Create a Pod Group:**
+
+To start using Gang Scheduling, create a Pod Group using the following command:
+
+```console
+$ kubectl apply -f manifests/quickstart-feature-examples/gang-scheduling/podgroup.yaml
```

Comment:
suggest displaying the yaml here, instead of referencing by path.

Reply:
addressed